rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/activityList.vue

```diff
@@ -801,6 +802,25 @@ export default defineComponent({
       }
       return 0;
     },
+
+    // Below code makes the progress bar reach closer to 100%
+    /* jobDurationPercentage(exec: Execution) {
+      if (
+        exec.job &&
+        exec.job.averageDuration &&
+        exec.dateStarted &&
+        exec.dateStarted.date
+      ) {
+        const now = moment();
+        const startDate = moment(exec.dateStarted.date);
+        const diff = now.diff(startDate);
+        const percentage = Math.floor((diff / exec.job.averageDuration) * 100);
```

Comment:
One heads-up, if it's the first time running a job, `averageDuration` isn't available. We shouldn't be showing 0 in this case.